spec/models/candidate_interface/volunteering_role_form_spec.rb

```diff
@@ -28,9 +28,8 @@
   describe '.build_all_from_application' do
```

Comment:
Ditto for the `update` spec in this file (line 95)

Reply:
Done in the commit about refactoring the `VolunteeringRoleForm` and spec.